base/setup/lib/utils/filesup.c

```diff
@@ -27,35 +27,32 @@
 static
 NTSTATUS
 SetupCreateSingleDirectory(
-    IN PCWSTR DirectoryName)
+    _In_ PCWSTR DirectoryName)
 {
+    NTSTATUS Status;
+    UNICODE_STRING PathName;
     OBJECT_ATTRIBUTES ObjectAttributes;
     IO_STATUS_BLOCK IoStatusBlock;
-    UNICODE_STRING PathName;
     HANDLE DirectoryHandle;
-    NTSTATUS Status;
 
-    if (!RtlCreateUnicodeString(&PathName, DirectoryName))
-        return STATUS_NO_MEMORY;
+    RtlInitUnicodeString(&PathName, DirectoryName);
 
     if (PathName.Length > sizeof(WCHAR) &&
         PathName.Buffer[PathName.Length / sizeof(WCHAR) - 2] == L'\\' &&
```

Comment:
nit: should this be `OBJ_NAME_PATH_SEPARATOR` ?

Reply:
It's the same, yes. I could use the define name here instead.